src/test/java/ru/ayaz/CommandExecutorTest.java

```diff
@@ -2,95 +2,61 @@
 
 import org.junit.Before;
 import org.junit.Test;
+import ru.ayaz.ru.ayaz.exceptions.InvalidUserCommandException;
 
-import java.util.HashMap;
-import java.util.Map;
-
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
 import static org.mockito.Mockito.*;
 
 public class CommandExecutorTest {
```

Comment:
CommandExecutor closes streams in first group of responsibilities and execute ignore command in second group of responsibilities. We have 50/50 distribution situation.  But in future, in my opinion, we will have to separate them again. So I agree that code should correspond to current conjuncture. I will combine them